CRM/Core/Page/Inline/Help.php

```diff
@@ -52,4 +53,12 @@ public function run() {
     }
   }
 
+  private function getAccessKey(): string {
+    $ua = strtolower($_SERVER['HTTP_USER_AGENT'] ?? '');
+    if (str_contains($ua, 'mac')) {
+      return '<span>CTRL</span>+<span>ALT</span>';
```

Comment:
Looks good. I'm just wondering now if we should take the opportunity to change `ALT` to `Option` on the mac condition, as that's the language mac keyboards almost always use (and would match what's documented as the keyboard combination here: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/accesskey)

Reply:
So would we simply change the word `ALT` above to `OPTION` (I no longer have a mac)

Reply:
Yes, exactly.